README

```diff
@@ -1 +1,67 @@
-Presto Manager: A tool for managing Presto clusters.
+# Presto Manager
+
+A tool for managing Presto clusters.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
```

Comment:
Make this a bulleted list (put each item on a line starting with `- ` (dash, space)).